middleware/csrf.go

```diff
@@ -0,0 +1,51 @@
+package middleware
+
+import (
+	"net/http"
+
+	"github.com/nikumar1206/puff"
+)
+
+type CSRFMiddlewareConfig struct {
```

Comment:
this is not how to document the struct. Here is example
// Regexp is the representation of a compiled regular expression.
// A Regexp is safe for concurrent use by multiple goroutines,
// except for configuration methods, such as Longest.
type Regexp struct {
    ...
}
the comments must be placed above the attribute as well.